### DISCUSSION TITLE
Kafka summit abstract

### DISCUSSION THREAD

**Discussion by astubbs** [confluentinc/parallel-consumer#68] [Show and tell] [answered]
Hi everyone! FYI I've submitted an abstract proposal for a Kafka summit talk introducing the project. I know it's a bit late (around 48 hours left to deadline), but if you have any suggestion for the abstract, let me know!

cc @JorgenRingen @mauricioszabo @ismarslomic 

Here’s my abstract on the Parallel Consumer intro: https://docs.google.com/document/d/1ygIuOD-V_ItScrMUeBipC9zhkJ2TYXx9aw7JISn3gcs/edit?usp=sharing

Feedback welcome! I still have time to edit it (it’s already submitted) =D
Introducing Confluent labs Parallel Consumer client
Description

Consuming messages in parallel is what Apache Kafka® is all about, so you may well wonder, why would we want anything else? It turns out that, in practice, there are a number of situations where Kafka’s partition-level parallelism gets in the way of optimal design.

This session will go over some of these types of situations that can benefit from parallel message processing with a single application instance (aka slow consumers or competing consumers), and then introduce the new Parallel Consumer labs project from Confluent, which can improve functionality and massively improve performance in such situations.

It will cover the
- Different ordering modes of the client
- Relative performance improvements
- Usage with other components like Kafka Streams
- An introduction to the internal architecture of the project
- How it can achieve all this in a reassignment friendly manner

https://confluentcommunity.slack.com/archives/CEJAEUSEQ/p1610654949002700

**Comment by ismarslomic** [comment, marked as answer]
Sounds like a very interesting talk, @astubbs! I guess you plan to explain ordering guarantees and offset commits in topic "Different ordering modes of the client"? If so, maybe topic could be renamed to "Different ordering modes and commit guarantees of the client"?
